news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: How advances in technology affected the osu! community
```

Comment:
```suggestion
title: "How advances in technology affected the osu! community"
```
i think the lack of quotes probably breaks something? better safe than sorry imo

Reply:
![image](https://github.com/ppy/osu-wiki/assets/24534022/4e1974be-792e-4ce5-94a8-4d7f20630b7b)
its fine bcs i was getting CI fails when quotes were present